docs/tutorial-run-hello-world.md

```diff
@@ -1,3 +1,29 @@
 # Running `hello-world.wasm` with Wasmtime
 
-... more coming soon
+## Installing Wasmtime
+
+The Wasmtime CLI can be installed on Linux and macOS with a small install
+script:
+
+```sh
+$ curl https://wasmtime.dev/install.sh -sSf | bash
+```
+
+Windows or otherwise interested users can download installers and binaries
+directly from the
+[GitHub Releases](https://github.com/bytecodealliance/wasmtime/releases) page.
```

Comment:
Could this link directly to the CLI installation docs as well? I think it's alright to inline a few of them, but we want to make sure it's easy to get mor einformation about installing if necessary.